### PR TITLE
docs: move router store selector StackBlitz example

### PR DESCRIPTION
**Pull request by cedricduffournet**
## PR Checklist

Please check if your PR fulfills the following requirements:

- [x] The commit message follows our guidelines: https://github.com/ngrx/platform/blob/master/CONTRIBUTING.md#commit
- [ ] Tests for the changes have been added (for bug fixes / features)
- [x] Documentation has been added / updated (for bug fixes / features)

## PR Type

What kind of change does this PR introduce?

```
[ ] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Build related changes
[ ] CI related changes
[x] Documentation content changes
[ ] Other... Please describe:
```

## What is the current behavior?
router store selectors StackBlitz example has direct link to StackBlitz

Closes #

## What is the new behavior?
router store selectors StackBlitz example use StackBlitz api

## Does this PR introduce a breaking change?

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: projects/ngrx.io/content/examples/router-store-selectors/src/app/router.selectors.ts [hunk 1]
```diff
@@ -0,0 +1,17 @@
+// #docregion routerSelectors
+import { getSelectors, RouterReducerState } from '@ngrx/router-store';
+import { createFeatureSelector } from '@ngrx/store';
+
+export const selectRouter = createFeatureSelector<RouterReducerState>('router');
```
Comment (timdeschryver) [inline projects/ngrx.io/content/examples/router-store-selectors/src/app/router.selectors.ts:5]: Using `getSelectors()` would also work if the default router configuration is used (`router` as key`). 

What if we add this line as a comment, and mention that this should only be required if the user deviates from the default config?

**Review by timdeschryver — COMMENTED** [review]
🏆

File: projects/ngrx.io/content/examples/router-store-selectors/src/app/router.selectors.ts (hunk 1 above)
Comment (cedricduffournet) [reply projects/ngrx.io/content/examples/router-store-selectors/src/app/router.selectors.ts]: Did it but not sure it's well written

File: projects/ngrx.io/content/examples/router-store-selectors/src/app/router.selectors.ts [hunk 2]
```diff
@@ -0,0 +1,18 @@
+// #docregion routerSelectors
+import { getSelectors, RouterReducerState } from '@ngrx/router-store';
+
+// `router` is used as default feature name. You can use the feature name
+// of you choice by creating a feature selector and pass it to getSelectors function
+// export const selectRouter = createFeatureSelector<RouterReducerState>('yourFeatureName');
```
Comment (timdeschryver) [inline projects/ngrx.io/content/examples/router-store-selectors/src/app/router.selectors.ts:6]:
```suggestion
// `router` is used as the default feature name. You can use the feature name
// of your choice by creating a feature selector and pass it to the `getSelectors` function
// export const selectRouter = createFeatureSelector<RouterReducerState>('yourFeatureName');
```

Comment (timdeschryver) [reply projects/ngrx.io/content/examples/router-store-selectors/src/app/router.selectors.ts:6]: Thanks!

**Review by timdeschryver — APPROVED** [review]
Nice !

**Review by brandonroberts — APPROVED** [review]

**Comment by brandonroberts** [comment]
Thanks @cedricduffournet!

**Pull request merged**